src/app/core/service/api/error-handler.ts

```diff
@@ -15,36 +15,31 @@ export class ErrorHandler {
 
   public handleError(err: any) {
     if (err.status === 0) {
-      let dialogRef = this.dialog.open(DialogComponent, {});
+      const dialogRef = this.dialog.open(DialogComponent, {});
       dialogRef.componentInstance.type = 'msg_refresh_dlg';
-    }
-    else if (err.status === 401) {
+    } else if (err.status === 401) {
       if (!this.dialog.openDialogs.length) {
         this.auth.logout();
-        let dialogRef = this.dialog.open(DialogComponent, {});
+        const dialogRef = this.dialog.open(DialogComponent, {});
         dialogRef.componentInstance.type = 'msg_auth_dlg';
       }
-    }
-    else if (err.status === 404) {
+    } else if (err.status === 404) {
       if (!this.dialog.openDialogs.length) {
-        let dialogRef = this.dialog.open(DialogComponent, {});
+        const dialogRef = this.dialog.open(DialogComponent, {});
         dialogRef.componentInstance.type = 'msg_404_dlg';
       }
-    } 
-    else if (err.status === 403) {
+    } else if (err.status === 403) {
       if (!this.dialog.openDialogs.length) {
-        let dialogRef = this.dialog.open(DialogComponent, {
+        const dialogRef = this.dialog.open(DialogComponent, {
```

Comment:
Rank B
The data of `dialogRef ` has a change, why did you use `const`

Reply:
In this case, `dialogRef ` just represent for  `this.dialog.open(DialogComponent, {})` and `dialogRef` will call to property `type` of `DialogComponent` 
=> it not reassigned for other value like example below.
```
const dialogRef = this.dialog.open(DialogComponent, {});
dialogRef = 'other value';
```